packages/rancher-monitoring/templates/crd-template/templates/jobs.yaml

```diff
@@ -49,8 +49,12 @@ spec:
           - -c
           - >
             echo "Applying CRDs...";
-            kubectl apply -f /etc/config/crd-manifest.yaml;
-            
+            mkdir -p /etc/crd-manifest;
```

Comment:
note: the fact that the initContainers still use the YAML files in `crd-manifest` means that this CRD chart today relies on the fact that the contents of `crd-manifest.tgz` exactly match that which was unfurled in the `crd-manifest/` directory.

Should keep a note of this in the future, I believe this might have been what you were alluding to in https://github.com/rancher/charts-build-scripts/pull/64#discussion_r741631747?

Reply:
no resolution needed for this comment, you can just resolve it with no changes.